docs/src/main/asciidoc/deploying-to-openshift-s2i.adoc

```diff
@@ -52,7 +52,7 @@ curl $URL/hello/greeting/quarkus
 [TIP]
 ====
 The `oc new-app` command above uses a builder image compatible with JDK 8.
-In order to create a JDK 11 native compatible image use `quay.io/quarkus/ubi-quarkus-native-s2i:{graalvm-version}-java11` as a builder image.
+In order to create a JDK 11 native compatible image use `quay.io/quarkus/ubi-quarkus-native-s2i:{graalvm-flavor}` as a builder image.
```

Comment:
This one doesn't feel right too.

Reply:
Right let's drop it. Search and replace was a bit too extreme apparently :).